doc/reference/tooling/tt_cli/multiple_instances.rst

```diff
@@ -0,0 +1,117 @@
+.. _tt-instances:
+
+Managing multiple instances
+===========================
+
+The ``tt`` utility can manage Tarantool applications that run on
+multiple instances. With a single ``tt`` call, you can:
+
+*   start an application on multiple instances (``tt start``)
+*   check the status of application instances (``tt status``)
+*   connect to a specific instance of an application (``tt connect``)
+*   stop a specific instance of an application or all its instances (``tt stop``)
+
+
+Application directory
+---------------------
+
+To create an application that runs on multiple instances, prepare its configuration
+in a directory. The directory name is used as the application's identifier.
+
+This directory should contain the following files:
+
+*   The application file named ``init.lua``.
+*   The instances configuration file ``instances.yml`` with the following format:
+
+    ..  code:: yaml
+
+        <instance_name1>:
+        <instance_name2>:
+        ...
```

Comment:
Does this example suggest that there are only instance names in the file and nothing else? I'd imagine something like this:
```
<instance_name1>:
  advertise_uri: 172.19.0.10:3301
  http_port: 8081
```
If I'm wrong and the file only contains instance names with semicolons, please make it clear.

Reply:
Only instance names for now.
A follow-up task on documenting available parameters once they are ready: https://github.com/tarantool/doc/issues/3156